src/services/user/services/AdminUsers.js

```diff
@@ -248,6 +252,19 @@ class AdminUsers {
 		}
 	}
 
+	async markUserAsDeleted(email, userId) {
+		if (email && userId) {
+			await this.app.service('accountModel').patch(null, {
```

Comment:
please call user service here and update the user model, not the account model

Reply:
done ;)

Reply:
done ;)